Content.Shared/Clothing/Components/ChameleonClothingComponent.cs

```diff
@@ -32,18 +32,26 @@ public sealed partial class ChameleonClothingComponent : Component
     /// </summary>
     [ViewVariables]
     public EntityUid? User;
+
+    /// <summary>
+    ///     Filter possible chameleon options by a tag in addition to WhitelistChameleon.
+    /// </summary>
+    [ViewVariables, DataField]
```

Comment:
```suggestion
    [DataField]
```
i think useless? ViesVariables already implemented into DataField iirc